src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -53,19 +54,23 @@ public CodeGenIL(IDictionary<string, IList> input, string filePath)
             writer = new StreamWriter(filePath);
         }
 
-        public IDictionary<string, IList> Emit(List<AssociativeNode> astList)
+        public (IDictionary<string, IList>, long) Emit(List<AssociativeNode> astList)
         {
             var compileResult = CompileAstToDynamicType(astList, AssemblyBuilderAccess.RunAndSave);
```

Comment:
so, later we'll wrap this some other timer? But this method now only returns a single result. I guess even though it's public, we're free to change it until this is more than a proof of concept.

Reply:
I will add and use an Internal Property instead.